pulsefire/ratelimiters.py

```diff
@@ -53,15 +53,17 @@ class RiotAPIRateLimiter(BaseRateLimiter):
     Parameters:
         proxy: URL of the proxy rate limiter.
         proxy_secret: Secret of the proxy rate limiter if required.
+        limiting_share: Value from 0 to 1. Rate limiter will only allow requests up to `bucket_max * limiting_share`.
     """
 
     _index: dict[tuple[str, int, *tuple[str]], tuple[int, int, float, float, float]] = \
         collections.defaultdict(lambda: (0, 0, 0, 0, 0))
 
-    def __init__(self, *, proxy: str | None = None, proxy_secret: str | None = None) -> None:
+    def __init__(self, *, proxy: str | None = None, proxy_secret: str | None = None, limiting_share: float = 1) -> None:
```

Comment:
While this parameter name was used on the predecessor package. A more proper and verbal parameter name would be `limit_perc`, please update it to that.